test/test_gpmsa/test_gpmsa_vector.C

```diff
@@ -343,17 +342,16 @@ int main(int argc, char ** argv) {
   proposalCovMatrix(4, 4)   = 0.6719 / 10.0;  // param 5
   proposalCovMatrix(5, 5)   = 0.4953 / scale;  // emulator precision
   proposalCovMatrix(6, 6)   = 0.4953 / scale;  // weights0 precision
-  proposalCovMatrix(7, 7)   = 0.4953 / scale;  // weights1 precision
-  proposalCovMatrix(8, 8)   = 0.6058 / scale;  // emulator corr str
-  proposalCovMatrix(9, 9)   = 7.6032e-04 / scale;  // emulator corr str
-  proposalCovMatrix(10, 10) = 8.3815e-04 / scale;  // emulator corr str
-  proposalCovMatrix(11, 11) = 7.5412e-04 / scale;  // emulator corr str
-  proposalCovMatrix(12, 12) = 0.2682 / scale;  // emulator corr str
-  proposalCovMatrix(13, 13) = 0.0572 / scale;  // emulator corr str
-  proposalCovMatrix(14, 14) = 1.3417 / scale;  // discrepancy precision
-  proposalCovMatrix(15, 15) = 0.3461 / scale;  // discrepancy corr str
-  proposalCovMatrix(16, 16) = 495.3 / scale;  // emulator data precision
-  proposalCovMatrix(17, 17) = 0.4953 / scale;  // observation error precision
+  proposalCovMatrix(7, 7)   = 0.6058 / scale;  // emulator corr str
```

Comment:
As long as we're fixing indexing... we should fix these comments later.  Truncation error precision should be off for this run, and we have 2 outputs, in which case we should follow the 5 non-hyper parameters with 2 emulator precision hyperparameters.  The "emulator precision" vs "weights precision" was a bit of confusion months ago on my part.

1 scenario and 5 uncertain parameters means 6 correlation strengths, and everything after that follows correctly too.